### DISCUSSION TITLE
Is there any fork for civitai helper that works in this?

### DISCUSSION THREAD

**Discussion by deagzzz** [lllyasviel/stable-diffusion-webui-forge#75] [Q&A]
Buttons for it doesnt seem to work

**Comment by aib0lit** [comment]
Try civitai-shortcut - it more complex in settings, but much more powerful

**Reply by deagzzz** [reply]
Thanks i'll try it. I just want the ability to go straight to civitai's page from a lora. That's literally the reason I'm holding up to switch to this entirely.

**Reply by aib0lit** [reply]
in civitai-shortcut you can view civitai lora page and information in build-in model browser, download example images (including user uploads), manipulate with images locally and send prompts to txt2ing and etc. :)

**Reply by deagzzz** [reply]
thats awesome! thanks man!

**Comment by JunJi-JiaYue** [comment]
Yes, there is. I use this one.
https://github.com/blue-pen5805/Stable-Diffusion-Webui-Civitai-Helper

**Comment by freecoderwaifu** [comment]
The zixaphir fork updated recently and has fixed buttons. It's the main fork listed in the Extensions downloader.

https://github.com/zixaphir/Stable-Diffusion-Webui-Civitai-Helper

**Comment by ostap667inbox** [comment]
With any fork of CivitaiHelper, I don't have Forge loading at all.
I see this error message while loading:
```
Traceback (most recent call last):
  File "C:\Users\itech\AppData\Local\Programs\Python\Python310\lib\threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "C:\Users\itech\AppData\Local\Programs\Python\Python310\lib\threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "C:\stable-diffusion-webui-forge\webui.py", line 84, in webui_worker
    app, local_url, share_url = shared.demo.launch(
  File "C:\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 1858, in launch
    self.validate_queue_settings()
  File "C:\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 1690, in validate_queue_settings
    raise ValueError("Progress tracking requires queuing to be enabled.")
ValueError: Progress tracking requires queuing to be enabled.
```
And then the process hangs on this line in the console:
`Model loaded in 7.2s (load weights from disk: 0.4s, forge load real models: 5.1s, load VAE: 0.5s, load textual inversion embeddings: 0.3s, calculate empty prompt: 0.9s).`

**Reply by ostap667inbox** [reply]
If I remove the `--no-gradio-queue` flag from the bat-file the problem goes away, but it is potentially fraught with problems with a large number of cards in the Extra Network tab - because of some problems with gradio when displaying the list of cards sometimes the process hangs with an error.
Also, the `--no-gradio-queue` flag is enabled in A1111 and Civitai Helper works in it.
